starcompactor/extractors/http.py

```diff
@@ -367,16 +184,16 @@ def traces(auth):
             LOG.debug('Invalid event %s', event)
         else:
             yield {
-                'uuid': instance['id'],
+                'INSTANCE_UUID': instance['id'],
                 'vcpus': flavor['vcpus'],
```

Comment:
I group vcpus, memory, and disk into one column/field called PROPERTIES. i.e. PROPERTIES is a json field. so that KVM and bare metal traces can share the same format, but different PROPERTIES contents. I use lowercase for the names of the properties. https://github.com/ChameleonCloud/starcompactor/blob/chi/starcompactor/formatters/csv.py#L18-L40